pkg/csi/service/common/constants.go

```diff
@@ -130,6 +130,18 @@ const (
 	// on which Snapshot feature is supported.
 	SnapshotSupportedVCenterPatch int = 3
 
+	// TKGsHASupportedVCenterMajor is the minimum major version of vCenter
+	// on which TKGsHA feature is supported.
+	TKGsHASupportedVCenterMajor int = 8
```

Comment:
remove these versions. refer to https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1462/files#r780576452